errbot/core_plugins/chatRoom.py

```diff
@@ -33,12 +33,15 @@ def callback_connect(self):
                     self.log.exception("Joining room %s failed", repr(room))
 
     def _join_room(self, room):
-        room_name = compat_str(room)
-        if room_name is not None:
-            room, username, password = (room_name, self.bot_config.CHATROOM_FN, None)
+        if isinstance(room, tuple):
```

Comment:
Checking for just tuples won't be sufficient here as a user could be using lists as well, which would give the same behaviour.